SwishExamples.playground/Contents.swift

```diff
@@ -0,0 +1,56 @@
+import PlaygroundSupport
+import Swish
+import Result
+
+PlaygroundPage.current.needsIndefiniteExecution = true
+
+// Let's say we have an endpoint, https://raw.githubusercontent.com/thoughtbot/Swish/master/Documentation/example.json
```

Comment:
Would it be possible to use playground markup instead of regular comments? Or does it automatically render these as Markdown?

Reply:
Good idea 👍